packages/platforms/accton/arm64/as4224/src/modules-5_6/arm64-accton-as4224-psu.c

```diff
@@ -166,6 +199,24 @@ static int as4224_psu_get_psu_count(void)
     return (status & 0x10) ? 1 : 2;
 }
 
+static int get_platform_id(void)
+{
+    int status;
+
+    status = as4224_psu_read_value(BOARD_INFO_REG_OFFSET);
+    if (status < 0)
+        return status;
+
+    if (status & 0x10)
+        return AS4224_52T;
+    else if (status & 0x20)
+        return AS4224_52T_DAC;
+    else if (status & 0x80)
+        return AS5114_48X;
+    else
+        return AS4224_52P;
```

Comment:
That would be 0x40?

Reply:
Code is revised, 0x40 is unknown platform. AS4224_52P is 0x00.